### ISSUE TITLE
Keyboard navigation unexpectedly throws user to the other column in stacked column chart when setting some value to null

### ISSUE THREAD

**Issue by maksim-pushkin-team-wrike** [highcharts/highcharts#17733]
#### Expected behaviour
When I have one of the values for chart set to null, I expect keyboard navigation to work in an expected way:
when press left/right arrow button it should lead focus to the other columns if there are such.
And when I press up/down arrow button it should lead focus to the other bars inside the same column if there are such

#### Actual behaviour
When I have a value in some column set to null, let's say, between the other not-null values and mean to navigate the column containing the null value by pressing up/down arrows, the focus gets thrown to the other columns, which is unexpected and also makes navigation a bit complicated.

#### Live demo with steps to reproduce
In this fiddle the first column (Arsenal) has null value.
https://jsfiddle.net/me9xab5r/4/

#### Product version
10.2.1

#### Affected browser(s)
probably, all

**Comment by maksim-pushkin-team-wrike** [comment]
This bug is also actual for the more natural case like in this demo: https://www.highcharts.com/demo/bar-stacked,the focus can be thrown unexpectedly to the neighbour column while navigation

**Comment by oysteinmoseng** [comment]
Thanks for reporting. This is a fairly general issue that applies to many chart types with the current keyboard navigation model. The logic should be rewritten to better handle cases like this.

A suboptimal workaround is to set `seriesNavigation.mode` to serialize, which will make it behave consistently, but only allow one dimensional navigation through each series: https://jsfiddle.net/dhp0nfj4/

**Comment by kamil-musialowski** [comment]
unstale, issue still present

**Comment by oysteinmoseng** [comment]
We will investigate if we can provide a quick fix for this, or a workaround. The longer term (2024, early 2025) plan involves a rewrite of how Highcharts handles keyboard navigation in general.

**Comment by marvin19** [comment]
**Workaround**: https://jsfiddle.net/maritavindedal/nhbfqe9o/
Created custom accessibility component to tweak keyboard navigation logic for stacked column chart. Included more null points than the original demo to make sure solution is flexible.
